realm-annotations/src/main/java/io/realm/annotations/RealmNamingPolicy.java

```diff
@@ -91,7 +91,7 @@
  *     </li>
  * </ol>
  * <p>
- * Note, that changing the internal name does <i>NOT</i> effect importing data from JSON. The JSON
+ * Note that changing the internal name does <i>NOT</i> effect importing data from JSON. The JSON
```

Comment:
Now I wonder if this is `affect` or `effect`.

I never know that.

Reply:
https://www.vocabulary.com/articles/chooseyourwords/affect-effect/

It should be affect I think :)